service/installer/helm/helm.go

```diff
@@ -232,7 +238,7 @@ func (i *HelmInstaller) checkHelmRelease(projectList []string) {
 
 		args := []string{"history", prj, "--output", "yaml", "--max", "1", "--namespace"}
 		if prj == "draughtsman" {
-			args = append(args, "default")
+			args = append(args, i.namespace)
```

Comment:
What is prj here? Do we still need the else if this is also draughtsman?

Reply:
Sorry for the dumb questions. I just forgot how any of this works :( or maybe I never knew!

So will this migrate the helm release to the draughtsman namespace when we deploy it? Do we need to remove any code afterwards?

If this does the migration and its fine in testing then I'm happy.

Reply:
Migration is quite complex, I need to delete draughtsman in the wrong namespace and should re-deploy in the `draguthsman` namespace. 

These changes rather remove the namespace hard-code parts (where it is installed), which means we don't need any code changes after migration.  

If this is not clear for you, please let me know.

Reply:
Thanks for the explanation I get this now. 👍